src/pages/product/components/vendor-preview-mode-modal/index.tsx

```diff
@@ -30,22 +31,35 @@ const VendorPreviewModeModal: React.FC<VendorPreviewModeModalProperties> = ({
 	return (
 		<>
 			<VendorDialog onClose={handleClose} open={isModalOpen}>
-				<DialogTitle sx={{ m: 1, p: 2 }} variant="playfairDisplayBold">
-					{t("ProductPage.payAttention")}
-				</DialogTitle>
-				<StyledCrossIconButton aria-label="close" onClick={handleClose}>
-					<CloseIcon />
-				</StyledCrossIconButton>
-				<DialogContent sx={{ m: 1, p: 3 }}>
+				<Box sx={{ display: "flex", justifyContent: "flex-end" }}>
+					<StyledCrossIconButton aria-label="close" onClick={handleClose}>
+						<CloseIcon />
+					</StyledCrossIconButton>
+				</Box>
+				<StyledDialogContent>
+					<StyledVendorPreviewIcon>
+						<VendorPreviewIcon />
+					</StyledVendorPreviewIcon>
+					<Typography
+						sx={{ fontSize: fontSizes.extraLarge, textAlign: "center" }}
+						variant="playfairDisplayBold"
+					>
+						{t("ProductPage.youAreInPreviewMode")}
+					</Typography>
 					<Typography
-						sx={{ fontSize: fontSizes.mediumLarge, lineHeight: "30px" }}
+						sx={{ color: colors.secondaryTextGray, textAlign: "center" }}
```

Comment:
In my opinion, we should use theme for all colors, font size, font weight from now.